README.md

```diff
@@ -0,0 +1,26 @@
+Usage:
+
+```
+import { RedisPubSub } from 'bgio-redis-pubsub'
+import { Server, Origins, SocketIO } from 'boardgame.io/server';
+import TicTacToe from './src/tic-tac-toe/game';
+import Chess from './src/chess/game';
+import redis from 'redis';
+import { PostgresStore } from 'bgio-postgres';
+ 
+const PORT = process.env.PORT || 8000;
+const redisConfig = { host: '192.168.8.50' };
+const pubClient = redis.createClient(redisConfig);
+const subClient = redis.createClient(redisConfig);
+const server = Server({
+  games: [TicTacToe, Chess],
```

Comment:
I’d keep this usage example as simple as possible, so just import a single game and call it `MyGame` or similar.

Reply:
Done.